CHANGELOG.md

```diff
@@ -7,7 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
 - Add terraform `v1.x` support
+- Add `aws_route_table` as dependencies to `aws_route` resources
```

Comment:
not change din this commit. please always update CHANGELOG in seperate commit to easier resolve conflicts